source/features/separate-draft-pr-gist-creation-button.css

```diff
@@ -0,0 +1,5 @@
+/* For the 1st button "Create public Gist" */
+button[value='1'] {
```

Comment:
This selector is too broad

Reply:
Yes, I thought so.
This `#new_gist > div > #gists+div >button[value="1"]` is good, right?

Reply:
One ID is good enough. If nothing better exists: `#gists + div button[value="1"]`

Reply:
I used `#gists + div button[value="1"]`. Thank you.

Reply:
Btw, there's a think I wanted to report some time now:
when running `npm run lint` I get:

```
Error: No files matching the pattern "'source/**/*.css'" were found.
    at C:\Users\Kostas\Documents\GitHub\refined-github\node_modules\stylelint\lib\standalone.js:212:12
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
ERROR: "lint:css" exited with 1.
```

Maybe the line `"lint:css": "stylelint 'source/**/*.css'",` in `package.json` should change somehow (its glob pattern?) ?
I am asking because in this case, the double-quotes in the selector above would trigger the `string-quotes / Expected single quotes` stylelint rule, but instead I get the error message.

Reply:
https://github.com/sindresorhus/refined-github/pull/4386